README.md

```diff
@@ -223,6 +223,40 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
 
 For iOS the XCode project needs to be generated with `CMake` as described [above](#ios) and added to your `xcworkspace`.
 
+## Supported Versions
+
+When using a React Native version in between the versions listed, use the previous React Native version. For example, React Native version 0.66 or 0.67 should use Babylon React Native versions that correspond to 0.65.
+
+| React Native | Babylon React Native |
+|---|---|
+| 0.64 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.65 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.69 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.70 | 1.5.0, 1.5.1 |
+
+3 NPM Packages are available: 1 mandatory for Windows or Android/iOS, 1 for Windows and 1 for Android/iOS.
```

Comment:
Suggestion (clarity):
`1 mandatory for Windows or Android/iOS` -> `1 mandatory for all platforms`